core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -654,7 +654,22 @@ public SELF withExposedPorts(Integer... ports) {
      * @param containerPort
      */
     protected void addFixedExposedPort(int hostPort, int containerPort) {
-        portBindings.add(String.format("%d:%d", hostPort, containerPort));
+        addFixedExposedPort(hostPort, containerPort, InternetProtocol.TCP);
```

Comment:
AFAIR if the protocol is not specified, it will use both, no? If so, this is a breaking change

Reply:
No @bsideup by default is TCP. Proof: https://docs.docker.com/config/containers/container-networking/

Reply:
👍